examples/async-with-routing/components/Layout.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export default class Layout extends Component {
```

Comment:
I suggest you to keep it but rename it to `App` and add some sort of header there, for example, the link to the source code like in the `real-world` example.